api/v1/helpers/verbs/doDelete.js

```diff
@@ -50,6 +53,8 @@ function doDelete(req, res, next, props) {
 
     // when a resource is deleted, delete all its associations too
     u.deleteAllAssociations(o, assocNames);
+    resultObj.retval = o.dataValues;
+    u.logAPI(req, resultObj);
```

Comment:
I don't like the mix of calling this function both as u.logAPI(...) and as logAPI(...) (line 36) in the same module--it's too confusing to the reader. Please standardize and call it one way across all API modules.